x-pack/plugins/security_solution/cypress/objects/rule.ts

```diff
@@ -180,6 +180,24 @@ export const newRule: CustomRule = {
   maxSignals: 100,
 };
 
+export const unmappedRule: CustomRule = {
+  customQuery: '*:*',
+  index: ['unmapped*'],
+  name: 'Rule with unmapped fields',
+  description: 'The new rule description.',
+  severity: 'High',
+  riskScore: '17',
+  tags: ['test', 'newRule'],
+  referenceUrls: ['https://www.google.com/', 'https://elastic.co/'],
```

Comment:
I would use http://example.com and https://example.com as that is reserved for developers by the IANA:
https://www.iana.org/domains/reserved